### PR TITLE
Mijn-7849 error handling

### PR DESCRIPTION
**Pull request by timvanoostrom**

### CODE REVIEW

File: src/server/config.ts [hunk 1]
```diff
@@ -591,3 +591,8 @@ export const securityHeaders = {
     require-trusted-types-for 'script'
   `.replace(/\n/g, ''),
 };
+
```
Comment (OscarBakker) [inline src/server/config.ts:594]: can be removed?

File: src/index.tsx [hunk 2]
```diff
@@ -45,8 +46,9 @@ Sentry.init({
   },
 });
 
-const sendToSentry = (error: Error, info: { componentStack: string }) => {
-  Sentry.captureException(error, {
+const sendToSentry = (error: Error, info: ErrorInfo) => {
+  Sentry.captureMessage(`Kritieke applicatie fout: ${error.message}`, {
+    level: 'fatal',
```
Comment (OscarBakker) [inline src/index.tsx:51]: what's the purpose of this?

**Review by OscarBakker — APPROVED** [review]
looks fine to me!

couple questions..

File: src/index.tsx (hunk 2 above)
Comment (timvanoostrom) [reply src/index.tsx:51]: https://docs.sentry.io/platforms/javascript/usage/set-level/ maar wat het doet in de Sentry omgeving moet ik nog even ondervinden :-)

File: src/server/config.ts (hunk 1 above)
Comment (timvanoostrom) [reply src/server/config.ts]: yes

**Pull request merged**